### ISSUE TITLE
Blocking image loading

### ISSUE THREAD

**Issue by SergeyTverye** [simonfrey/faster-pageload-web-extensions#15]
Hi, Simon! I'm not sure mb my settings in Firefox are wrong, but the console says that Faster Pageload is blocking the loading of all images on this page https://cloudflare-quic.com/ if lazy loading enable in extension settings

Screen: https://i.comss.pics/2020/11/25/1111111111111333333333333333333.jpg

Can you check it? Thanks!

**Comment by simonfrey** [comment]
Yes this is indeed a bug. I will check how to fix it. Will not be able to do this before christmas though. If anyone else would jump onto this issue would be great.

**Comment by simonfrey** [comment]
Until than you could disable the image lazy loading to prevent the issue

**Comment by simonfrey** [comment]
Should be fixed with 1.8.5 :D Please check
